dex/testing/firo/harness.sh

```diff
@@ -258,23 +260,23 @@ ${CLI} ${GAMMA_CLI_CFG} "\$@"
 EOF
 chmod +x "./gamma"
 
-cat > "./reorg" <<EOF
-#!/usr/bin/env bash
-set -x
-echo "Disconnecting beta from alpha"
-sleep 1
-./beta disconnectnode 127.0.0.1:${ALPHA_LISTEN_PORT}
-echo "Mining a block on alpha"
-sleep 1
-./mine-alpha 1
-echo "Mining 3 blocks on beta"
-./mine-beta 3
-sleep 2
-echo "Reconnecting beta to alpha"
-./beta addnode 127.0.0.1:${ALPHA_LISTEN_PORT} onetry
-sleep 2
-EOF
-chmod +x "./reorg"
+# cat > "./reorg" <<EOF
+# #!/usr/bin/env bash
+# set -x
+# echo "Disconnecting beta from alpha"
+# sleep 1
+# ./beta disconnectnode 127.0.0.1:${ALPHA_LISTEN_PORT}
+# echo "Mining a block on alpha"
+# sleep 1
+# ./mine-alpha 1
+# echo "Mining 3 blocks on beta"
+# ./mine-beta 3
+# sleep 2
+# echo "Reconnecting beta to alpha"
+# ./beta addnode 127.0.0.1:${ALPHA_LISTEN_PORT} onetry
+# sleep 2
+# EOF
+# chmod +x "./reorg"
```

Comment:
What's up with this?

Reply:
It's fine!
I had the script apart and 'all over the floor' while trying to figure out what had changed to make some rpcs not work. Then when putting stuff back together I did not incude this as I never used it. I will put it back in.